src/software/geom/angular_velocity.h

```diff
@@ -1,6 +1,11 @@
 #include "software/geom/angle.h"
 
-// We also use variables of type 'Angle' to represent angular velocities, since they
-// are essentially represented the same. This typedef allows us to refer to Angles as
-// AngularVelocities, which makes the interfaces more intuitive.
-typedef Angle AngularVelocity;
+/**
+ * We also use variables of type 'Angle' to represent angular velocities, since they
+ * are essentially represented the same. This typedef allows us to refer to Angles as
+ * AngularVelocities, which makes the interfaces more intuitive.
+ * @note Not all methods of Angle class make sense for AngularVelocity!
+ * E.g. Angle::clamp does not make sense in the context of AngularVelocity, as 360 deg/s
+ * is different from 0 deg/s, but 360 deg is the same as 0 deg.
+ */
+using AngularVelocity = Angle;
```

Comment:
Maybe a good future task difficulty 3 ticket is to create an AngularVelocity class that wraps an Angle

Reply:
Good idea. Added #3093